docs/pages/installation/wsl.md

```diff
@@ -66,11 +66,11 @@ Once Docker is installed, the next step will be to install a Kubernetes cluster.
 In this how-to, [K3d](https://github.com/rancher/k3d) will be used as it relys only on Docker.
 
 ```bash
-curl -s https://raw.githubusercontent.com/rancher/k3d/master/install.sh | bash
-k3d create
-export KUBECONFIG="$(k3d get-kubeconfig --name='k3s-default')"
+curl -s https://raw.githubusercontent.com/rancher/k3d/main/install.sh | bash
+k3d cluster create
+export KUBECONFIG="$(k3d kubeconfig get 'k3s-default')"
```

Comment:
@leecalcote, please review this part, looks good to me. @thegagne did a fabulous job!